src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -62,7 +62,6 @@ limitations under the License.
             <include cluster="General Commissioning" client="false" server="true" clientLocked="true" serverLocked="true">
                 <requireCommand>SetRegulatoryConfig</requireCommand>
             </include>
-            <include cluster="Power Source Configuration" client="false" server="false" clientLocked="true" serverLocked="false"></include>
```

Comment:
Essentially, because Power Configuration got removed in this commit https://github.com/project-chip/connectedhomeip/commit/e9960d4a577a6afb361f54ec13875ae47a922865#diff-3918846a50017bc6b4593fc1afef2ce31e97060b45e517b4ce7c752b3d0c3db3.

I was assuming that Power Configuration is the same as Power Source Configuration (amplified by the description which mentions "power source(s)").

Reply:
It's not the same thing....

Reply:
I see, re-added Power Source/Power Source Configuration.